api/weather/utils.go

```diff
@@ -13,5 +13,5 @@ func buildurl(zipcode int, apikey string) string {
 }
 
 func kelvinToF(k float64) float64 {
-	return (k-273.15)*(9/5) + 32
+	return (((k - 273.15) * 9) / 5) + 32
```

Comment:
Is there a weather conversion package we can use, so we don't have to handle the math ourselves?

Reply:
Maybe we can try [this](https://github.com/martinlindhe/unit) unit conversion package?